Detections/SecurityEvent/StartStopHealthService.yaml

```diff
@@ -40,6 +41,34 @@ query: |
   ) on TargetLogonId
   | project TimeGenerated, Computer, Account, TargetAccount, IpAddress,TargetLogonId, ObjectServer, ObjectType, ObjectName
   | extend timestamp = TimeGenerated, HostCustomEntity = Computer, AccountCustomEntity = Account, IPCustomEntity = IpAddress
+  ),
+  (WindowsEvent
+  | where EventID ==4656 and EventData has_all ("HealthService", "SERVICE OBJECT",  "SC Manager")
+  | extend ObjectServer = tostring(EventData.ObjectServer)
+  | extend ObjectType = tostring(EventData.ObjectType)
+  | extend ObjectName = tostring(EventData.ObjectName)
+  | where isnotempty(ObjectServer) and isnotempty(ObjectType) and isnotempty(ObjectName)
+  | where ObjectServer =~ "SC Manager" and ObjectType =~ "SERVICE OBJECT" and ObjectName =~ "HealthService"
+  | extend TargetLogonId = tostring(EventData.TargetLogonId)
+  | extend Account =  strcat(tostring(EventData.SubjectDomainName),"\\", tostring(EventData.SubjectUserName))
+  | extend TargetAccount = strcat(EventData.TargetDomainName,"\\", EventData.TargetUserName)
+  | extend IpAddress = tostring(EventData.IpAddress)
+    // Comment out the join below if the SACL only audits users that are part of the Network logon users, i.e. with user/group target pointing to "NU."
+  | join kind=leftouter (
+    WindowsEvent
+  | where EventID == 4624
+  |extend TargetLogonId = tostring(EventData.TargetLogonId)
+  | extend ObjectServer = tostring(EventData.ObjectServer)
```

Comment:
You don't need to extend those fields in this side of the join. They don't exist in 4624 and you can see they are not used later. The second copy of a field is called <name>1, so ObjectServer1.